imaplugin/src/main/java/com/kaltura/playkit/plugins/ima/ExoPlayerWithAdPlayback.java

```diff
@@ -612,6 +613,16 @@ public void setVolume(float volume) {
         adPlayer.setVolume(volume);
     }
 
+    public void setSurfaceAspectRatioResizeMode(PKAspectRatioResizeMode resizeMode, boolean isUpdateResizeMode) {
+        if (resizeMode == null) {
```

Comment:
This feature is fine. If the video frame is already it covering the whole width and if 'aspectRatio' is set to `fixedWidth` then nothing will happen to the frame. Same in case of other aspect ratios.